tasks/validate.yml

```diff
@@ -17,7 +17,9 @@
       Elasticsearch process' maximum number of open file descriptors is not set
       high enough (i.e. to more than 64000). For more info, see:
       https://www.elastic.co/guide/en/elasticsearch/guide/current/_file_descriptors_and_mmap.html
-  when: elasticsearch_process_res.json.nodes.values().0.process.max_file_descriptors | int < 64000
+  when: >
+    (elasticsearch_process_res.json.nodes.values() | 
```

Comment:
This line break looks wired. Can you try to come up with something better?

Reply:
Single line looks the best in this case, but it exceeds the print margin.